ui/main/src/app/store/effects/user.effects.ts

```diff
@@ -39,9 +44,9 @@ export class UserEffects {
     checkUserApplication: Observable<UserActions> = createEffect(() =>
         this.actions$.pipe(
             ofType(AuthenticationActionTypes.AcceptLogIn),
-            switchMap((action: AcceptLogIn) => {
+            switchMap((action: AcceptLogInAction) => {
```

Comment:
remove the parameter action as suggested by sonar [here](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=3092&id=org.lfenergy.operatorfabric%3Aoperatorfabric-core&open=AYEfdJxOl4nUV2p2c4FG)